docs/reference/permission.rst

```diff
@@ -1,26 +1,125 @@
 .. _permission:
 
-Permission
-##########
+Permissions
+###########
 
+*Cliquet* provides a mechanism to handle authorization on the stored objects.
 
-Redis
-=====
+Glossary
+========
 
-.. autoclass:: cliquet.permission.redis.Redis
+Authorization isn't complicated, but requires the introduction of a few terms
+so that explanations are easier to follow:
 
+**Object**:
```

Comment:
We already have a glossary in reference folder. Shouldn't this be merged there ?